po/cs.po

```diff
@@ -0,0 +1,141 @@
+# Czech translation for flatseal package.
+# Copyright (C) 2020 THE flatseal'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+#
+# Daniel Rusek <mail@asciiwolf.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-02-02 22:58-0300\n"
+"PO-Revision-Date: 2020-02-03 14:58+0100\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Daniel Rusek <mail@asciiwolf.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.2.4\n"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:3
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:3 src/aboutDialog.ui:14
+msgid "Flatseal"
+msgstr "Flatseal"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:9
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:8
+msgid "A permissions manager for Flatpak"
+msgstr "Správce oprávnění pro Flatpak"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:10
+msgid "com.github.tchx84.Flatseal"
+msgstr ""
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:10 src/aboutDialog.ui:12
+msgid ""
+"Flatseal is a graphical utility to review and modify basic permissions from "
+"your Flatpak applications."
+msgstr ""
+"Flatseal je grafický nástroj pro prohlížení a úpravu základních oprávnění "
+"vašich Flatpak aplikací."
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:29
+msgid "Martin Abente Lahaye"
+msgstr "Martin Abente Lahaye"
+
+#: src/aboutDialog.ui:11
+msgid "Copyright © 2020 - Martín Abente Lahaye"
+msgstr "Copyright © 2020 - Martín Abente Lahaye"
+
+#: src/view.ui:36
+msgid "No applications found."
+msgstr "Nenalezeny žádné aplikace."
+
+#: src/window.ui:41
+msgid "Reset"
+msgstr "Resetovat"
+
+#: src/window.ui:47
+msgid "Reset this application permisions"
+msgstr "Resetovat oprávnění této aplikace"
+
+#: src/model.js:35
+msgid "Access network"
+msgstr "Přístup k síti"
+
+#: src/model.js:40
+msgid "Access inter-process communications"
+msgstr "Přístup k meziprocesové komunikaci"
+
+#: src/model.js:45
+msgid "Access X11 windowing system"
+msgstr "Přístup k okennímu systému X11"
+
+#: src/model.js:50
+msgid "Access X11 windowing system (as fallback)"
+msgstr "Přístup k okennímu systému X11 (jako rezerva)"
+
+#: src/model.js:55
+msgid "Access Wayland windowing system"
+msgstr "Přístup k okennímu systému Wayland"
+
+#: src/model.js:60
+msgid "Access PulseAudio sound server"
+msgstr "Přístup ke zvukovému serveru PulseAudio"
+
+#: src/model.js:65
+msgid "Access D-Bus system bus (unrestricted)"
+msgstr "Přístup k systémové sběrnici D-Bus (neomezený)"
+
+#: src/model.js:70
+msgid "Access D-Bus session bus (unrestricted)"
+msgstr "Přístup ke sběrnici sezení D-Bus (neomezený)"
+
+#: src/model.js:75
+msgid "Access Secure Shell agent"
+msgstr "Přístup k agentu Zabezpečeného Shellu"
```

Comment:
Can you leave "Secure Shell" ? Since it's a proper name.

Reply:
I can, however "Zabezpečený Shell" is commonly used in most of our (Czech) language translations. Also, "k agentu Secure Shellu" sounds even worse in my opinion.

Reply:
You're the expert, is fine then.

Reply:
Well, to be honest, I don't like the translated version either. :-) Unfortunately, both "Zabezpečený Shell" and "Secure Shell" sound bad in our language, but the first one is at least commonly used in translations.